### PR TITLE
Allow users to upload an avatar feature

### PR DESCRIPTION
**Pull request by sebastiancaraballo**
**Trello board reference:**

- https://trello.com/c/jx6HB7Ia

**Description:**
- Allow users to upload avatars.

### CODE REVIEW

File: spec/uploaders/avatar_uploader_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  end
+
+  after do
+    AvatarUploader.enable_processing = false
+    uploader.remove!
+  end
+
+  context 'the thumb version' do
+    it 'scales down avatar to be exactly 64 by 64 pixels' do
+      expect(uploader.thumb).to have_dimensions(64, 64)
```
Comment (glm4) [inline spec/uploaders/avatar_uploader_spec.rb:21]: I think resize_to_fit preserves the original aspect ratio, so the resulting image could have lower dimensions for either width or height.

File: spec/uploaders/avatar_uploader_spec.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    end
+
+    it 'has the correct format' do
+      expect(uploader).to be_format('png')
+    end
+  end
+
+  context 'the normal version' do
+    it 'scales down avatar to be exactly 512 by 512 pixels' do
+      expect(uploader.normal).to have_dimensions(512, 512)
```
Comment (glm4) [inline spec/uploaders/avatar_uploader_spec.rb:31]: same here for the dimensions.

**Review by glm4 — APPROVED** [review]
Small comments :)

File: app/uploaders/avatar_uploader.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+class AvatarUploader < CarrierWave::Uploader::Base
+  include CarrierWave::MiniMagick
+
+  def store_dir
+    "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+  end
+
+  def default_url(*)
```
Comment (GuilleLeopold) [inline app/uploaders/avatar_uploader.rb:8]: Why did you need to define this default_url?

**Review by mdornel — APPROVED** [review]
